client/api_object.go

```diff
@@ -469,12 +470,51 @@ func (c *client) GetObjectPolicy(ctx context.Context, bucketName, objectName str
 }
 
 // ListObjects return object list of the specific bucket
-func (c *client) ListObjects(ctx context.Context, bucketName string, opts types.ListObjectsOptions) (types.ListObjectsResult, error) {
+func (c *client) ListObjects(ctx context.Context, bucketName, maxKeys, startAfter, continuationToken, delimiter, prefix string, opts types.ListObjectsOptions) (types.ListObjectsResult, error) {
```

Comment:
suggest to put  the new parameters to option,  beacause they are not necessary to run this API . And add comments about the fields to explain the meaning

for example
```
type ListObjectsOptions struct {
	ShowRemovedObject bool
         maxKey uint64
	// StartAfter start listing lexically at this
	// object onwards, object onwards
      	StartAfter string
	// Only list objects with the prefix
	Prefix string
        delimiter string
}
```

Reply:
fixed